services/prometheus/prometheus.go

```diff
@@ -346,46 +346,17 @@ func (svc *Service) marshalConfig() ([]byte, error) {
 			scrapeConfigForQANAPI2(s.MR),
 		)
 
-		cfg.AlertingConfig.AlertmanagerConfigs = append(cfg.AlertingConfig.AlertmanagerConfigs, &config.AlertmanagerConfig{
-			ServiceDiscoveryConfig: config.ServiceDiscoveryConfig{
-				StaticConfigs: []*config.Group{{
-					Targets: []string{"127.0.0.1:9093"},
-				}},
-			},
-			Scheme:     "http",
-			PathPrefix: "/alertmanager/",
-			APIVersion: config.AlertmanagerAPIVersionV2,
-		})
+		conf, err := svc.parseAlertmanagerConfigFromURL("http://127.0.0.1:9093/alertmanager/")
```

Comment:
Let's also revert this change as it seems to be changing logic, and generally bigger that other changes

Reply:
Reverted.